pkg/operator2/auth.go

```diff
@@ -1,15 +1,21 @@
 package operator2
 
 import (
+	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	configv1 "github.com/openshift/api/config/v1"
 )
 
 func (c *authOperator) handleAuthConfig() (*configv1.Authentication, error) {
 	auth, err := c.authentication.Get(globalConfigName, metav1.GetOptions{})
+
 	if err != nil {
-		return nil, err
+		if !errors.IsNotFound(err) {
+			return nil, err
+		}
+		// did not find the object, use default
+		auth = defaultAuthenticationConfig()
```

Comment:
I think you need to create the default object (before attempting to update it's status 🙂 )